server/src/main/java/org/elasticsearch/http/HttpStats.java

```diff
@@ -62,6 +63,14 @@ public List<ClientStats> getClientStats() {
         return this.clientStats;
     }
 
+    public static HttpStats merge(HttpStats first, HttpStats second) {
+        return new HttpStats(
```

Comment:
This is pretty straightforward, but probably worth a unit test

Reply:
will do!